Lib/unittest/mock.py

```diff
@@ -1927,16 +1927,16 @@ def mock_add_spec(self, spec, spec_set=False):
 
 class MagicProxy(object):
     def __init__(self, name, parent):
-        self.name = name
-        self.parent = parent
+        self._mock_name = name
+        self._mock_parent = parent
```

Comment:
Not sure this is related. Please can you add a unit test to show why it's needed?

Reply:
I've just did a deeper check and it looks like leaving it as it was before doesn't break anything - it's just a matter of conforming to the naming. So I believe we can leave it as it is.